### ISSUE TITLE
AVT 1 - React Expandable Tile has a DAP violation 

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#3374]
## Environment
macOS Mojave version 10.14.5
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Carbon v10 - React
DAP - July 2019 Ruleset

## Detailed Description
Run DAP on the Expanded Tile Component and the following violation is present:
<img width="931" alt="Screen Shot 2019-07-11 at 4 22 50 PM" src="https://user-images.githubusercontent.com/21676914/61087502-6f3e8d80-a3fb-11e9-93ff-4588e2d87785.png">

**Comment by jnm2377** [comment]
@carbon-design-system/design should the text color or background color be updated for this? And what token would it be?

**Comment by tw15egan** [comment]
@jnm2377 I'm about to push up a fix for this, the error was coming because the below-the-fold text is set to `opacity: 0` when it is collapsed. I just set it to also have a `visibility: hidden` so it is no longer trying to be read.
